ancillary/local-config-service/src/main/resources/config/loyalty-service.yaml

```diff
@@ -19,7 +19,7 @@ spring:
     security:
       protocol: ${KAFKASECURITYPROTOCOL:PLAINTEXT}
     consumer:
-      group-id: ${KAFKA_CONSUMER_GROUP_ID:loyaltyservice}
```

Comment:
Agree with this convention and this is how I started the project; however, Key Vault doesn't support underscores, which is extremely frustrating! Anyway, I voted for https://feedback.azure.com/d365community/idea/8b46339b-f624-ec11-b6e6-000d3a4f0da0 (hopefully, you can too). We really need key Vault to support common naming conventions for Java (and many other) workloads.